new_movies_tracker.py

```diff
@@ -85,16 +90,16 @@ def __process_torrent(self, torrent, plugin):
         movie = Movie(torrent)
         key = movie.orig_name
         if key in self.loaded_movies_db:
-            print('[!exst] {}'.format(movie))
+            logger.info('[!exst] {}'.format(movie))
             return
         if self.min_seeders and movie.seeders < self.min_seeders:
-            print('[!seed] {}'.format(movie))
+            logger.info('[!seed] {}'.format(movie))
             return
         if self.min_size and movie.size < self.min_size:
-            print('[!size] {}'.format(movie))
+            logger.info('[!size] {}'.format(movie))
             return
 
-        print('      + {}'.format(movie))
+        logger.info('      + {}'.format(movie))
```

Comment:
nit: remove leading whitespaces . It was ok for `print`, but will be ugly when printed by logger.